src/Medidata.ZipkinTracer.Core.Collector/Properties/AssemblyInfo.cs

```diff
@@ -37,7 +37,7 @@
 // You can specify all the values or you can default the Build and Revision Numbers 
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyVersion("1.3.0")]
-[assembly: AssemblyFileVersion("1.3.0")]
-[assembly: AssemblyInformationalVersion("1.3.0")]
+[assembly: AssemblyVersion("1.4.0")]
```

Comment:
@bvillanueva-mdsol I think you are right, I tend to use the following pattern quite a lot:

- One **GlobalAssemblyInfo.cs** file in the root folder (the same folder where the **.sln** file is) with the `AssemblyCompany`, `AssemblyProduct`, `AssemblyCopyright`, `AssemblyTrademark`, `AssemblyCulture`, `AssemblyConfiguration` and `ComVisible` attributes
- Either the `AssemblyVersion`, `AssemblyFileVersion` and `AssemblyInformationalVersion` attributes in the **GlobalAssemblyInfo.cs** (if you want all your projects to have the same version number in the solution), or a dedicated **AssemblyVersionInfo.cs** file in the root if you want to have subgroups with different versions in the solution - but this is the exceptional case
- Every project contains their own respective **AssemblyInfo.cs** file with the `AssemblyTitle`, `AssemblyDescription` and `Guid` attributes only
- The **GlobalAssemblyInfo.cs** _linked_ to the projects and the link is located under the _Properties_ node in the Solution Explorer

With this pattern it's quite easy to control the solution's administrative aspects and it is flexible enough to handle any exceptional cases (like having all but one assembly to have the same version numbers)